server/pbenchinacan/load_keycloak.sh

```diff
@@ -17,25 +17,23 @@
 # "http://localhost:8080/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
 # env variable.
 
-KEYCLOAK_BASE_IMAGE=${KEYCLOAK_BASE_IMAGE:-"quay.io/keycloak/keycloak:latest"}
+KEYCLOAK_BASE_IMAGE=${KEYCLOAK_BASE_IMAGE:-"images.paas.redhat.com/pbench/pbenchinacan-keycloak:base"}
```

Comment:
`base` is fine for now, but as soon as feasible we should change that to something which identifies in a meaningful way the image which we are caching, a la `postgresql-13` or `7.13.3` (for Elasticsearch).